src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -1236,6 +1258,16 @@ private BlockPos getNodeMiningPosition(BlockPos blockToMine)
             {
                 return blockToMine;
             }
+            final Point2D parentPos = buildingMiner.getCurrentLevel().getRandomNode().getParent();
+            if(parentPos != null && buildingMiner.getCurrentLevel().getNode(parentPos) != null
+                    && buildingMiner.getCurrentLevel().getNode(parentPos).getStyle() == Node.NodeType.SHAFT)
+            {
+                final BlockPos ladderPos = buildingMiner.getLadderLocation();
+                return new BlockPos(
+                        ladderPos.getX() + buildingMiner.getVectorX() * OTHER_SIDE_OF_SHAFT
+                        , buildingMiner.getCurrentLevel().getDepth()
```

Comment:
Why are we doing this stupid thing with the commas? They always go end the end of the line when splitting?